obspy/io/gcf/__init__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+obspy.io.gcf - Guralp Compressed Format read support for ObsPy
+==============================================================
+This module provides read support for GCF waveform data and header info.
+Most methods are based on info from Guralp site
+http://geophysics.eas.gatech.edu/GTEQ/Scream4.4/GCF_Specification.htm
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org) & Ran Novitsky Nof
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+    
+Reading
+-------
+Similar to reading any other waveform data format using :mod:`obspy.core`
+The format will be determined automatically.
+
+Writing
```

Comment:
Yes, just remove those 4-5 lines about writing. The first line says "This module provides read support for", that's sufficient I think.